packages/synapse-react-client/src/components/SchemaDrivenAnnotationEditor/SchemaDrivenAnnotationEditor.tsx

```diff
@@ -60,16 +61,18 @@ export type SchemaDrivenAnnotationEditorProps = {
   schemaId?: string
   /** May be used to directly provide a JSON Schema to use for the form */
   validationSchema?: JSONSchema7
-  /** Optionally supply a ref to the form to handle submission externally with `formRef.current.submit()`. If provided, the editor will not render its own submit UI. */
+  /** Optionally supply a ref to the form to handle submission externally with `formRef.current.submit()`. */
   formRef?: React.RefObject<RJSF>
-  /** Provide live input validation. This can cause major performance degradation. */
+  /** Provide live input validation. This can cause performance degradation. By default, liveValidate will be true if an entity with a schema and existing annotations is being edited */
   liveValidate?: boolean
   /** Invoked after a successful form submission */
   onSuccess?: () => void
   /** If defined and formRef is not supplied, shows a 'Cancel' button and runs this effect on click */
   onCancel?: () => void
   /** Passes new form data upon each change to the form */
   onChange?: (annotations: Record<string, unknown>) => void
+  /** If true, the editor will not render its own submit UI. */
+  hideActions?: boolean
```

Comment:
This behavior was based on supplying a `formRef`, but really it should be a separate prop